packages/frontend/core/src/pages/index.tsx

```diff
@@ -152,13 +152,15 @@ export const Component = () => {
           position: 'fixed',
           left: '50%',
           top: '50%',
+          zIndex: 1,
         }}
       >
         <Menu
           rootOptions={{
             open: true,
           }}
           items={<UserWithWorkspaceList />}
+          noPortal
```

Comment:
Why noPortal here?

Reply:
This portal will cover the dialog, it is zindex higher than dialog.